src/systemtask/WakeLock.cpp

```diff
@@ -0,0 +1,23 @@
+#include "systemtask/WakeLock.h"
+
+using namespace Pinetime::System;
+
+WakeLock::WakeLock(SystemTask& systemTask) : systemTask {systemTask} {
```

Comment:
should we add a destructor to release the lock if held?

Reply:
Destructor is in the header file, I could move it if that's better. As it's a 1 liner I just left it in there

Reply:
I overlooked that the destructor is already here. Bummer, didn't find a bug :grin: 

I always try to put the implementations in the cpp file as leaving the implementations in the `h` or `hpp` files the one definition rule sometimes is violated and breaks compiles. But as this isn't the case here, I have no objections

Reply:
I don't really know best C++ practices, if I should move it I'm very happy to

Reply:
no, leave it. If it ever becomes a problem the compiler will loudly and accurately complain about it